packages/zoe/src/internal-types.js

```diff
@@ -97,7 +97,8 @@
  * @property {() => BrandKeywordRecord} getBrands
  * @property {() => Object} getTerms
  * @property {() => boolean} hasShutdown
- * @property {() => void} shutdown
+ * @property {() => void} terminate
+ * @property {(error: string) => void} terminateOnFailure
```

Comment:
Is there a reason to switch to `terminate`? Regardless, we should be consistent in the language. `hasShutdown` shouldn't be here.

Reply:
@erights and I talked about it, and the conclusion was that the successful and unsuccessful shutdown paths should be similarly named. @warner pointed out that shutdown is both noun and verb, while terminate is only a verb. So the rule is that (internally) we're using `terminate` and `terminateOnFailure` consistently. 

The verb for ending the participation of an individual seat should be different from that for a vat, so the user-facing terminology wasn't changed.

I'll rename hasShutdown to hasTerminated. It's not visible outside of zoe.